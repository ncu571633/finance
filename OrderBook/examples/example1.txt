# Example orders file

BUY GFD 1000 10 order1
SELL GFD 900 10 order2