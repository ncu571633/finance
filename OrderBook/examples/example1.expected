TRADE order1 1000 10 order2 900 10
